README.md

```diff
@@ -280,55 +330,114 @@ make kurtosis.clean
 
 ```shell
-make spec-test
+make grafana-up
 ```
 
-Or only run those of a specific config with:
+After that, open [`http://localhost:3000/`](http://localhost:3000/) in a browser.
+The default username and password are both `admin`.
 
-```shell
-make spec-test-config-`config`
+To stop the containers run `make grafana-down`. For cleaning up the metrics data, run `make grafana-clean`.
 
-# Some examples
-make spec-test-config-mainnet
-make spec-test-config-minimal
-make spec-test-config-general
+## Benchmarks
+
+Several benchmarks are provided in the `/bench` directory. They are all standard elixir scripts, so they can be run as such. For example:
+
+```bash
+mix run bench/byte_reversal.exs
 ```
 
-Or by a single runner in all configs, with:
+Some of the benchmarks require a state or blocks to be available in the db. For this, the easiest thing is to run `make checkpoint-sync` so an anchor state and block are downloaded for mainnet, and optimistic sync starts. If the benchmark requires additional blocks, maybe wait until the first chunk is downloaded and block processing is executed at least once.
 
-```shell
-make spec-test-runner-`runner`
+Some need to be executed with `--mode db` in order to not have the store replaced by the application. This needs to be added at the end, like so:
 
-# Some examples
-make spec-test-runner-ssz_static
-make spec-test-runner-bls
-make spec-test-runner-operations
+```bash
+mix run <script> --mode db
 ```
 
-The complete list of test runners can be found [here](https://github.com/ethereum/consensus-specs/tree/dev/tests/formats).
+A quick summary of the available benchmarks:
 
-If you want to specify both a config and a runner:
+- `deposit_tree`: measures the time of saving and loading an the "execution chain" state, mainly to test how much it costs to save and load a realistic deposit tree. Uses benchee. The conclusion was very low (the order of μs).
+- `byte_reversal`: compares three different methods for byte reversal as a bitlist/bitvector operation. This concludes that using numbers as internal representation for those types would be the most efficient. If we ever need to improve them, that would be a good starting point. 
+- `shuffling_bench`: compares different methods for shuffling: shuffling a list in one go vs computing each shuffle one by one. Shuffling the full list was proved to be 10x faster.
+- `block_processing`: Builds a fork choice store with an anchor block and state. Uses the next block available to apply `on_block`, `on_attestation` and `on_attester_slashing` handlers. Runs these handlers 30 times. To run this, at least 2 blocks and a state must be available in the db. It also needs you to set the slot manually at the beginning of an epoch. Try it for the slot that appeared when you ran checkpoint sync (you'll see in the logs something along the lines of `[Checkpoint sync] Received beacon state and block slot=9597856`)
+- `multiple_block_processing`:
```

Comment:
We should add a description for this (similar to `block_processing` but with a wider range of slots) or maybe remove the file.

Reply:
Missed it, thanks for the comment!